UnityProject/Assets/Scripts/Doors/DoorTrigger.cs

```diff
@@ -22,16 +22,20 @@ public void Start()
 
         public override void Interact(GameObject originator, Vector3 position, string hand)
         {
-            AccessRestrictions AR = GetComponent<AccessRestrictions>();
             
-            if(doorController.IsOpened)
+            // Close the door if it's open
+            if(doorController.IsOpened && allowInput)
             {
-                doorController.CmdTryClose();
+                PlayerManager.LocalPlayer.GetComponent<PlayerNetworkActions>().CmdTryCloseDoor(gameObject);

```

Comment:
You could make this shorter by defining:
`PlayerNetworkActions pna = PlayerManager.LocalPlayer.GetComponent<PlayerNetworkActions>()`

Then you could just reuse:
`pna.CmdTryCloseDoor(gameObject)`
for example.

This also prevents your lines being seperated into two lines on automatic cleaning.

Reply:
Editied my comment